client_test.go

```diff
@@ -21,6 +21,14 @@ func TestClientAPIName(t *testing.T) {
 	}
 }
 
+func TestClientAPIDescription(t *testing.T) {
+	cs := NewClient("ENDPOINT", "KEY", "SECRET")
+	req := &ListAPIs{}
+	desc := cs.APIDescription(req)
+	if desc != "lists all available apis on the server" {
+		t.Errorf("APIDescription of listApis is wrong, got %q", desc)
+	}
+}
```

Comment:
[nitpick]: "\n" missing there, I think.

Reply:
@chrisglass the lib does it for you there: https://golang.org/src/log/log.go#L170